app/lldb_ruby.py

```diff
@@ -0,0 +1,8 @@
+def __lldb_init_module(debugger, dict):
+  debugger.HandleCommand('command script add -f lldb_ruby.rbp rbp')
+
```

Comment:
It doesn’t even state the rationale :-/

https://www.python.org/dev/peps/pep-0008/#blank-lines